.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+Fixes # .
```

Comment:
Is this implying that we should always open an issue to go with any PR? I think most of our PR's generally aren't directly addressing an issue.